smithy-model/src/test/resources/software/amazon/smithy/model/transform/rename-shape-test-model.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
For this integration test, I'd like to see metadata getting renamed (e.g., a suppression) and tests for virtually every kind of shape and reference -- structures, lists, maps, sets, unions, services, operations, etc.

Reply:
It looks like this is missing validation for updating some relationships:

* service -> operation
* service -> resource
* resource -> identifier
* resource -> operation